### PR TITLE
Pot random item drop

### LINKED ISSUE #46: Breakable Item
Create an object that can be destroyed which drops an item.

### PR DESCRIPTION
**Pull request by Dezerack**
Closes #46

### CODE REVIEW

File: Assets/Scripts/Weapons/ProjectileGun/Bullet.cs [hunk 1]
```diff
@@ -21,9 +21,9 @@ private void OnTriggerEnter2D(Collider2D collide)
     {
         if (collide.gameObject.tag != "Bullet")
         {
-            EnemyHealth enemyHealth = collide.gameObject.GetComponent<EnemyHealth>();
-            enemyHealth.Damage(Damage);
-            Destroy(collide.gameObject);
+            NonPlayerHealth nonPlayerHealth = collide.gameObject.GetComponent<NonPlayerHealth>();
+            nonPlayerHealth.Damage(Damage);
+            // Destroy(collide.gameObject);
```
Comment (BrandonPacewic) [inline Assets/Scripts/Weapons/ProjectileGun/Bullet.cs:26]: Does this need to be commented out still?

Comment (Galaxy25) [reply Assets/Scripts/Weapons/ProjectileGun/Bullet.cs:26]: It doesn't need to exist anymore. Also, the bullet needs to be destroyed at the end of the collide. @Dezerack

**Review by Galaxy25 — DISMISSED** [review]

File: Assets/Scripts/RandomItemDrop.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+// SPDX-License SPDX-License-Identifier: Apache-2.0
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RandomItemDrop : MonoBehaviour
+{
+    // you will probably want to move things around I just have it how I learned / how I stole it from a different
+    // project
```
Comment (BrandonPacewic) [inline Assets/Scripts/RandomItemDrop.cs:11]: Should I worry about changing anything? It all looks good to me.

File: ProjectSettings/ProjectVersion.txt [hunk 3]
```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2021.3.16f1
-m_EditorVersionWithRevision: 2021.3.16f1 (4016570cf34f)
+m_EditorVersion: 2021.3.11f1
+m_EditorVersionWithRevision: 2021.3.11f1 (0a5ca18544bf)
```
Comment (BrandonPacewic) [inline ProjectSettings/ProjectVersion.txt:2]: @Dezerack Can I get you to update the unity version?

**Review by BrandonPacewic — CHANGES_REQUESTED** [review]

File: Assets/Scripts/RandomItemDrop.cs (hunk 2 above)
Comment (Dezerack) [reply Assets/Scripts/RandomItemDrop.cs:11]: I just thought you had a way of organizing things but I didnt remember so I just used what I had you dont have to worry about changing things.

Comment (BrandonPacewic) [reply Assets/Scripts/RandomItemDrop.cs:11]: Looks fine to me so I'm going to remove this comment.

**Comment by BrandonPacewic** [comment]
Looks like this branch is on unity version `2021.3.17f1`. Going to bump it back down to what is currently on main and make a separate pr for this.

**Review by BrandonPacewic — APPROVED** [review]
Looks good to me! @Galaxy25 I believe that you will need to approve again since files have changed sense your review.

**Comment by Galaxy25** [comment]
@BrandonPacewic Reapproved, the code doesn't break anything

**Comment by BrandonPacewic** [comment]
Incredible work! 🚀

**Pull request merged**